apps/meteor/client/components/GenericModal.tsx

```diff
@@ -14,8 +14,8 @@ type GenericModalProps = RequiredModalProps & {
 	title?: string | ReactElement;
 	icon?: ComponentProps<typeof Icon>['name'] | ReactElement | null;
 	confirmDisabled?: boolean;
-	onCancel?: () => void;
-	onClose: () => void;
+	onCancel: () => void;
```

Comment:
Are you sure you need these changes? It's going to impact a lot of components

Reply:
just changed it back to optional...